pytensor/graph/basic.py

```diff
@@ -970,6 +970,136 @@ def applys_between(
     )
 
 
+def truncated_graph_inputs(
+    outputs: Sequence[Variable], conditions: Optional[Collection[Variable]] = None
+) -> List[Variable]:
+    """Get the conditional subset for outputs.
+
+
+    Conditional subset is a minimal set of variables required to compute outputs.
+
+    Parameters
+    ----------
+    outputs : Collection[Variable]
+        Variable to get conditions for
+    conditions : Optional[Collection[Variable]]
+        Additional conditions to assume, by default None
+
+    Returns
+    -------
+    List[Variable]
+        Variables required to compute ``outputs``
```

Comment:
This description is incorrect, no? In the case of no conditions, the outputs are returned.

Reply:
if you have no conditions you only need outputs so say what outputs are, why not?